actioncable/lib/action_cable/remote_connections.rb

```diff
@@ -19,6 +19,12 @@ module ActionCable
   # This will disconnect all the connections established for
   # <tt>User.find(1)</tt>, across all servers running on all machines, because
   # it uses the internal channel that all of these servers are subscribed to.
+  #
+  # You can also use the remote connection to forcibly unsubscribe a connection
+  # from a specific stream. For example:
+  #
+  #   subscription_identifier = "{\"channel\":\"ChatChannel\", \"chat_id\":1}"
```

Comment:
What about accepting Hash as an argument? IMO, JSON looks counterintuitive here. Instead we can do smth like:

```ruby
ActionCable.server.remote_connections.where(current_user: User.find(1)).unsubscribe_from(ChatChannel, chat_id: 1)

# or
ActionCable.server.remote_connections.where(current_user: User.find(1)).unsubscribe_from(channel: "ChatChannel", chat_id: 1)
```